resources/views/entities/components/_post_layouts.blade.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+* @var \App\Models\MiscModel $model
+* @var \App\Models\Entity $entity
+* @var \App\Models\Post $post
+* @var \Illuminate\Database\Eloquent\Collection $pinnedNotes
+*/
+?>
+<div class="post-{{ $post->id }} entity-note-{{ $post->id }} entity-note-position-{{ $post->position }} post-position-{{ $post->position }}@if (isset($post->settings['class']) && $campaign->boosted()) {{ $post->settings['class'] }}@endif" data-visibility="{{ $post->visibility_id }}" data-position="{{ $post->position }}">
+    <div class="box box-solid post entity-note" id="post-{{ $post->id }}" style="background-color:transparent">
```

Comment:
We don't need the box layout, as this is only for text/posts and being removed. Please replace it with the same layout as used by organisation members :-) It should have 0 margin/padding on the sides and look more like this 
![image](https://github.com/owlchester/kanka/assets/3263440/15834b2f-ec22-44ff-88b8-47384f32f77c)